src/CTA.WebForms2Blazor/ControlConverters/HyperLinkControlConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using HtmlAgilityPack;
+
+namespace CTA.WebForms2Blazor.ControlConverters
+{
+    public class HyperLinkControlConverter : ControlConverter
+    {
+        private Dictionary<String, String> _hyperLinkControlsMap = new Dictionary<string, string>() 
+            {["navigateurl"] = "href", ["cssclass"] = "class"};
+        private readonly string _name = "a";
+        
+        public HyperLinkControlConverter() : base()
+        {
+            
+        }
+
+        public override HtmlNode Convert2Blazor(HtmlNode node)
+        {
+            string newAttributes = "";   
+            foreach (HtmlAttribute attr in node.Attributes)
+            {
+                if (_hyperLinkControlsMap.ContainsKey(attr.Name))
+                {
+                    string newAttr = _hyperLinkControlsMap[attr.Name] + "=" + attr.Value + " ";
```

Comment:
May want to make this into a template string like the one below, also this seems like it may be reused, maybe a function in the base class that takes in some number of control attribute maps and returns the formatted string would be good for this